SevenZip.Tests/SevenZip.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;net45;net462</TargetFrameworks>
```

Comment:
Why is this target changed from .NET Standard 2.0 to Core 2.1, while the SevenZip project itself remains Standard 2.0?

Reply:
it is only one way how I can run tests with not net45 and net462 frameworks
I found it at https://github.com/nunit/docs/wiki/.NET-Core-and-.NET-Standard#why-cant-my-tests-target-net-standard